index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html style=" background-color:aqua;
+text-align:center;
+" >
+    <head>
+    
```

Comment:
please, remove the spaces for clean code, spaces make your code not clean

Reply:
- [ ] Kindly make sure you are more detailed and constructive in your feedback.
`hint: You can suggest online documentation about any specific point that need to be improved`

Reply:
kindly , could you explain to me more about this issue?

Reply:
> kindly , could you explain to me more about this issue?

I wanted to say, you have to make sure you give an authentic reason of the changes you suggest.